ic.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+const assert = require('assert');
+const fs = require('fs');
+const rp = require('request-promise-native');
+const username = process.env.CIRCLE_TOKEN;
+const password = '';
+const auth = 'Basic ' + new Buffer(username + ':' + password).toString('base64');
+
+var projectName = JSON.parse(fs.readFileSync('./package.json')).name;
+//Override project name in the command line
+if(process.argv.length === 3) projectName = process.argv[2]
```

Comment:
What do you think of `const projectName = process.argv[2] || require('./package.json').name`?